tests/phpunit/tests/rest-api/rest-block-type-controller.php

```diff
@@ -243,7 +243,12 @@ public function test_get_item_invalid() {
 		$this->assertNull( $data['editor_style'] );
 		$this->assertNull( $data['style'] );
 		$this->assertSameSets( array(), $data['provides_context'] );
-		$this->assertSameSets( array(), $data['attributes'] );
+		$this->assertSameSets(
```

Comment:
```suggestion
		$this->assertSameSetsWithIndex(
```

:1234: Also, add the ticket number to the docblocks for each of the modified tests in this file. (They're outside the diff so I can't add a suggestion, sorry.)